src/pages/user/Home/HomePage.jsx

```diff
@@ -1,7 +1,40 @@
+import { CarouselChefs } from '@/components/CarouselChefs/CarouselChefs';
+import { CarouselDishes } from '@/components/CarouselDishes/CarouselDishes';
+import Hero from '@/components/Hero';
+import Overview from '@/components/Overview';
+import { route } from '@/constants';
+import { usePopularChefs, usePopularDishes } from '@/hooks';
 import { Main } from '@/shared/Main/Main';
 
 const HomePage = () => {
-  return <Main>Home Page</Main>;
+  const {
+    data: popularDishes = [],
+    //isLoading: popularDishesIsLoading,
+    //error: popularDishesError,
+  } = usePopularDishes();
+  const {
+    data: popularChefs = [],
+    //isLoading: popularChefsIsLoading,
+    //error: popularChefsError,
+  } = usePopularChefs();
+
+  console.log(popularChefs);
```

Comment:
Probably you just forgot to delete `console.log`

Reply:
Thx.